tests/Type/DateTimePickerTypeTest.php

```diff
@@ -86,9 +86,9 @@ public function testSubmitMatchingDateFormat(): void
             'html5' => false,
         ]);
 
-        static::assertSame('8:02 PM', $form->getViewData());
+        static::assertSame("8:02\u{202F}PM", $form->getViewData());
```

Comment:
I was also looking into it now while solving https://github.com/sonata-project/form-extensions/issues/521

I think better would be to not use `\IntlDateFormatter::NONE` above?

Reply:
using `'format' => 'h:mm a',` works fine

Reply:
Removing `\IntlDateFormatter::NONE` would give a different result `'Jun 3, 2018, 8:02 PM'`

Reply:
anyway I guess we can just keep this fix :+1: